### PR TITLE
feat: added eligibility and mandatory criteria to emails

### PR DESCRIPTION
**Pull request by IgnatG**
### Changelog:
- included the answers to the mandatory and eligibility criteria in the emails
- removed the re-tries for eStore api calls
- added a validation for deals/facilities that have mocked IDs

### CODE REVIEW

File: trade-finance-manager-api/src/constants/email-template-ids.js [hunk 1]
```diff
@@ -6,7 +6,9 @@ const BSS_DEAL_MIA_RECEIVED = '882bb325-b19e-4d18-b35e-f09ed787db7c';
 const GEF_DEAL_MIA_RECEIVED = '9a5a900d-624d-4fb6-9f99-5de191442e07';
 const ISSUED_FACILITY_RECEIVED = '56ad0bd0-abf7-4e29-a6db-986d8287969b';
 const BSS_DEAL_SUBMIT_CONFIRMATION = 'a6d4832e-01f3-4077-a571-b25ad9992533';
-const GEF_DEAL_SUBMIT_CONFIRMATION = 'dae1e902-5dc9-4ec4-aad7-9a161c96146f';
+// const GEF_DEAL_SUBMIT_CONFIRMATION = 'dae1e902-5dc9-4ec4-aad7-9a161c96146f';
+// TODO: replace after Caroline tests it
```
Comment (Zainzzkk) [inline trade-finance-manager-api/src/constants/email-template-ids.js:10]: ticket for this?

File: trade-finance-manager-api/src/v1/helpers/generate-mandatory-criteria-string.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-restricted-syntax */
```
Comment (Zainzzkk) [inline trade-finance-manager-api/src/v1/helpers/generate-mandatory-criteria-string.js:1]: not urgent - can you add a comment later about what this function does

**Review by Zainzzkk — APPROVED** [review]
LGTM

File: trade-finance-manager-api/src/constants/email-template-ids.js (hunk 1 above)
Comment (IgnatG) [reply trade-finance-manager-api/src/constants/email-template-ids.js:10]: we won't need a ticket. I'm going to revert it before deployment to prod

File: trade-finance-manager-api/src/v1/helpers/generate-mandatory-criteria-string.js (hunk 2 above)
Comment (IgnatG) [reply trade-finance-manager-api/src/v1/helpers/generate-mandatory-criteria-string.js:1]: good shout

**Pull request merged**